komodo/data/enable.in

```diff
@@ -60,7 +72,6 @@ export _PRE_KOMODO_LD_LIBRARY_PATH="$LD_LIBRARY_PATH"
 export LD_LIBRARY_PATH=komodo_prefix/lib:komodo_prefix/lib64${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
 
 export _PRE_KOMODO_PS1="${PS1:-}"
-export PS1="(${KOMODO_RELEASE}) ${PS1:-}"
```

Comment:
Why are we removing `export PS1`? (best console and environment variable)

Reply:
wooops